### PR TITLE
postgresql_publication: add ability to specify schemas to add to publication

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.postgresql/issues/709

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

File: tests/integration/targets/postgresql_publication/tasks/postgresql_publication_initial.yml [hunk 1]
```diff
@@ -495,6 +495,144 @@
...
+    postgresql_publication: 
+      <<: *pg_parameters
+      name: test_schema_pub
+      tables_in_schema: '{{ test_schema }}'
+
+  - assert:
+      that:
+      - result is changed
+      - result.queries == ["CREATE PUBLICATION \"test_schema_pub\" FOR TABLES IN SCHEMA \"{{ test_schema }}\""]
+      - result.schemas == ["acme_schema"]
```
Comment (hunleyd) [inline tests/integration/targets/postgresql_publication/tasks/postgresql_publication_initial.yml:539]: why hardcoded 'acme_schema' and not the `{{ test_schema }}` var?
```
- result.schema == [test_schema]
```

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: tests/integration/targets/postgresql_publication/tasks/postgresql_publication_initial.yml (hunk 1 above)
Comment (Andersson007) [reply tests/integration/targets/postgresql_publication/tasks/postgresql_publication_initial.yml]: i had a feeling that using too many vars in a file scope in such cases is a bit overkill in general
but yeah, let's make it consistent. done, PTAL

**Review by hunleyd — APPROVED** [review]

**Comment by Andersson007** [comment]
@hunleyd thanks for reviewing, @commodis thanks for the feature request!

**Pull request merged**